app/components/provider_interface/individual_condition_status_review_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<span class="govuk-caption-l"><%= application_choice.application_form.full_name %></span>
+<h1 class="govuk-heading-l"><%= page_heading %></h1>
+
+<%= render ProviderInterface::OfferSummaryComponent.new(
+  application_choice: application_choice,
+  course_option: application_choice.current_course_option,
+  conditions: form_object.conditions,
+  course: application_choice.current_course,
+) %>
```

Comment:
This component looks like the thing we want to use when compared to the prototype. There's one issue: when the course has full and part time options, the change link renders next to the full time/part time row